app/scripts/templates/partial/coppa-age-input.mustache

```diff
@@ -0,0 +1,4 @@
+<div class="input-row">
+  <label class="label-helper"></label>
+  <input type="number" id="age" placeholder="{{#t}}How old are you?{{/t}}" min="0" max="150" value="{{ age }}" {{#shouldFocusAge}}autofocus{{/shouldFocusAge}} required />
```

Comment:
Yeah "max" is only useful if form controls are used. To keep it simple I'm just going to set it to 999, unless we want to be really strict about this.